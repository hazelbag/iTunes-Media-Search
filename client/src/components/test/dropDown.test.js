import React from 'react';
import DropDown from '../dropDown'
import renderer from 'react-test-renderer'

test('Link renders correctly', () => {
    const tree = renderer
      .create(<DropDown />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });