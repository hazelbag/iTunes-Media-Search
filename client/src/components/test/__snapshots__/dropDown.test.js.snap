// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<div
  className="drop-background dropdown-button"
>
  <div
    className="dropdown"
    onKeyDown={[Function]}
  >
    <button
      aria-expanded={false}
      aria-haspopup={true}
      aria-label={null}
      className="dropdown-toggle btn btn-secondary"
      onClick={[Function]}
      type="button"
    >
      Search 
      <span
        aria-label="magnifying glass"
        role="img"
      >
        üîç
      </span>
    </button>
    <div
      aria-hidden={true}
      className="dropdown-menu"
      role="menu"
      tabIndex="-1"
    >
      <a
        className="dropdown-item"
        href="/music"
        onClick={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        Music
      </a>
      <a
        className="dropdown-item"
        href="/musicVideo"
        onClick={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        Music Videos
      </a>
      <a
        className="dropdown-item"
        href="/movies"
        onClick={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        Movies
      </a>
      <a
        className="dropdown-item"
        href="/tvShow"
        onClick={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        TV Shows
      </a>
    </div>
  </div>
</div>
`;
